docs/integrations/ai-engines/neuralforecast.mdx

```diff
@@ -1,8 +1,190 @@
 ---
-title: NeuralForecast
+title: Nixtla's NeuralForecast Integration with MindsDB
 sidebarTitle: NeuralForecast
 ---
 
-<Note>
-This page is a work in progress.
-</Note>
+Nixtla’s NeuralForecast provides a diverse array of neural forecasting models, prioritizing their ease of use and resilience. These models encompass a spectrum of options, including traditional networks like MLP and RNNs, as well as cutting-edge innovations such as NBEATS, NHITS, TFT, and various other architectural approaches.
+
+You can learn more about its features [here](https://nixtla.github.io/neuralforecast/).
+
+## How to bring NeuralForecast Models to MindsDB
+
+Before creating a model, you will need to create an ML engine for NeuralForecast using the `CREATE ML_ENGINE` statement:
+
+```sql
+CREATE ML_ENGINE neuralforecast
+FROM neuralforecast;
+```
+
+Once the ML engine is created, we use the `CREATE MODEL` statement to create the NeuralForecast model in MindsDB.
+
+```sql
+CREATE MODEL model_name
+FROM data_source
+  (SELECT * FROM table_name)
+PREDICT column_to_be_predicted
+ORDER BY date_column
+GROUP BY column_name, column_name, ...
+WINDOW 12 -- model looks back at sets of 12 rows each
+HORIZON 3 -- model forecasts the next 3 rows
+USING ENGINE = 'neuralforecast';
+```
+
+To ensure that the model is created based on the NeuralForecast engine, include the `USING` clause at the end.
+
+## Example
+
+Let's go through an example of how to use Nixtla's NeuralForecast with MindsDB to forecast monthly expenditures based on historical data.
+
+Please note that before using the NeuralForecast engine, you should create it from the MindsDB Cloud editor with the below command:
```

Comment:
maybe we can change the `MindsDB Cloud Editor` in case someone is using it through python, dbeaver etc